### ISSUE TITLE
how support LLFF  Mip-NeRF 360 dataset?

### ISSUE THREAD

**Issue by goometasoft** [donydchen/mvsplat#1]
thank your mvsplat , i like it very much !

how support LLFF  Mip-NeRF 360 dataset?

**Comment by donydchen** [comment]
Hi @goometasoft, thanks for your interest in our work. 

To support other datasets, one key thing is the camera parameter. MVSplat uses OpenCV-style camera-to-world matrices and normalised camera intrinsics. By organising the datasets in such formats, it should be able to use our pre-trained weights.

Another thing is about view selection. Since MVSplat leverages the cost volume to fuse multi-view information, we assume that the input context views should have enough overlap, e.g., at least 40% to 60%. 

We have written data pre-processing scripts and data loaders for DTU and Mip-NeRF 360 in our experiments, data from both datasets works well with our RE10K pretrained weights. But we have not well-cleaned those parts of the codes, and we will release them once we finish organising them in the future. Besides, pixelSplat, which is the code base of MVSplat, has also shared some scripts on how to use data in COLMAP format at https://github.com/dcharatan/pixelsplat/issues/17, you might want to look that up for some more reference.

**Comment by XLR-man** [comment]
Hi, sorry for my poor knowledge:
for the colmap format dataset, how can i use the code in (https://github.com/dcharatan/pixelsplat/issues/17) to transfer dataset in .torch files?

**Comment by Fanjunyi55** [comment]
> 你好，抱歉我了解不多：关于 colmap 格式数据集，我该如何使用 （[dcharatan/pixelsplat#17](https://github.com/dcharatan/pixelsplat/issues/17)） 中的代码来传输 .torch 文件中的数据集？

您好，请问您实现运行自己的数据集了吗？
